astronomer/providers/amazon/aws/example_dags/example_s3.py

```diff
@@ -21,14 +34,43 @@
 
     waiting_for_s3_key = S3KeySensorAsync(
         task_id="waiting_for_s3_key",
-        bucket_key="sample_key.txt",
-        wildcard_match=False,
-        bucket_name="sample-bucket",
+        bucket_key=S3_BUCKET_KEY,
+        wildcard_match=WILDCARD_KEY_FALSE,
+        bucket_name=S3_BUCKET_NAME,
     )
 
     check_if_wildcard_exists = S3KeySensorAsync(
         task_id="check_if_wildcard_exists",
-        bucket_key="test*",
-        wildcard_match=True,
-        bucket_name="sample-bucket",
+        bucket_key=S3_BUCKET_WILDCARD_KEY,
```

Comment:
Does the file looking for pre exists or do we upload it from the dag? ideally we need to make it self sufficient dag so upload also has to be done from dag.

Reply:
[Done](https://github.com/astronomer/astronomer-providers/pull/86/commits/236024be56d984bdcb8c54a328597f43abbb3e14)